project2/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="card">
+        <div class="content">
+            <p class="moviename">Her</p>
+            <p class="rating">rating: 8.0/10</p>
+            <p class="description">R | 2h 6min | Drama, Romance, Sci-Fi | 10 January 2014 (USA)</p>
+            <p class="description">In a near future, a lonely writer develops an unlikely relationship with an operating
+                system
+                designed to meet his every need.</p>
+            <p><a href="https://www.youtube.com/watch?v=WzV6mXIOVl4">WATCH TRAILER</a></p>
```

Comment:
Text should not be in ALL CAPS for decorative purposes in the HTML; if you want to make the text for this link ALL CAPS, you should instead use CSS (`text-transform: uppercase;`).
```suggestion
            <p><a href="https://www.youtube.com/watch?v=WzV6mXIOVl4">Watch Trailer</a></p>
```